src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -96,45 +83,42 @@ public IntakeSubsystem(WPI_TalonFX motorOuterAxle,
     // Methods
 
     /**
-     * Manually sets the speed of the inner and outer axle motors.
+     * Manually sets the speed of the motor
      *
      * @param speed
      */
     public void setSpeed(double speed) {
-        motorInnerAxle.set(speed);
-        motorOuterAxle.set(speed);
+        motor.set(speed);
     }
 
     /**
-     * Spins motors inwards and updates motor state.
+     * Spins motor inwards and updates motor state.
      * Runs if Intake is extended which is when the solenoid is retracted.
      */
     public void intakeIn() {
         if (intakeSolenoidState == RETRACT) {
-            motorOuterAxle.set(INTAKE_IN_SPEED);
-            motorInnerAxle.set(INTAKE_IN_SPEED);
+            motor.set(INTAKE_IN_SPEED);
             intakeMotorState = IN;
         }
     }
 
     /**
-     * Spins motors outwards and updates motor state.
+     * Spins motor outwards and updates motor state.
      * Runs if Intake is extended which is when the solenoid is retracted.
      */
     public void intakeOut() {
         if (intakeSolenoidState == RETRACT) {
```

Comment:
This is the same as isIntakeExtended method, can you implement that instead for easier readability